### ISSUE TITLE
Alpha loss in (EC-)SAC different from the original paper

### ISSUE THREAD

**Issue by ChengshuLi** [tensorflow/agents#202]
Based on the [original paper](https://arxiv.org/pdf/1812.05905.pdf), alpha loss should be 
```
alpha_loss = tf.exp(self._log_alpha) * tf.stop_gradient(-log_pi - self._target_entropy)
```
instead of
https://github.com/tensorflow/agents/blob/8e8cc2e67b4026824885520cb4026c62b6e326e2/tf_agents/agents/sac/sac_agent.py#L515-L516

Is it intentional to use `log(alpha)` instead of `alpha` in the alpha loss function?

Thanks a lot! Appreciate your help.

**Comment by kuanghuei** [comment]
Yes. In the code that the paper authors released, they actually use log(alpha) instead of alpha in loss function: https://github.com/rail-berkeley/softlearning/blob/master/softlearning/algorithms/sac.py#L205

 Although I can't say for sure, I think it helps training through shaping the loss landscape.
